app/controllers/queue_items_controller.rb

```diff
@@ -29,12 +42,13 @@ def destroy
 
   private
 
-  def queue_order
-    current_user.queue_items.count + 1
+  def item_params
+    p = params.require(:queue_item).permit(:user_id, :video_id)
```

Comment:
Using 1 letter variables such like this is a little bit dangerous, for instance `p` is actually a ruby method which you have overwritten now.

My advice - try to avoid using confusing variable names.